packages/@aws-cdk/aws-eks/lib/fargate-cluster.ts

```diff
@@ -13,6 +14,14 @@ export interface FargateClusterProps extends ClusterOptions {
    *            selectors will be created if this is left undefined.
    */
   readonly defaultProfile?: FargateProfileOptions;
+  /**
+   * KMS secret for envelope encryption for Kubernetes secrets.
+   *
+   * @default - By default, Kubernetes stores all secret object data within etcd and
+   *            all etcd volumes used by Amazon EKS are encrypted at the disk-level
+   *            using AWS-Managed encryption keys.
+   */
+  readonly secretsEncryptionKey?: kms.IKey;
```

Comment:
Since `ClusterProps` extends `ClusterOptions`, we can just move `ClusterProps.secretsEncryptionKey` to `ClusterOptions.secretsEncryptionKey`.

Reply:
Sure, i'll update that.

Reply:
Sounds good. This is completed.

Reply:
👍